### PR TITLE
IPC-171 & IPC-170 rounded corners payment provider icon

### PR DESCRIPTION
**Pull request by danicretu**
Added rounded corners to payment provider app icons in `PaymentComponentView` & `BankSelectionBottomSheet` items.

### CODE REVIEW

File: health-sdk/sdk/src/main/res/layout/ghs_item_payment_provider_app.xml [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
+    android:background="@drawable/ghs_selectable_payment_provider_background"
+    >
 
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/ghs_payment_provider_app"
-        style="@style/GiniHealthTheme.Widget.Button.OutlinedButton.ListItem"
-        android:layout_width="0dp"
+    <include layout="@layout/ghs_payment_provider_selector_button"
+        android:id="@+id/ghs_selector_layout"
+        android:layout_width="match_parent"
```
Comment (abolfazlimahdi) [inline health-sdk/sdk/src/main/res/layout/ghs_item_payment_provider_app.xml:11]: I would recommend not using match_parent inside a contraintlayout. it does not behave normally. Also, this layout would work perfectly fine with a FrameLayout :-)

**Review by abolfazlimahdi — APPROVED** [review]
I added just one comment. The commit messages are also not conforming to the Gini's commit message style. Please consider using them. Thanks :)

File: health-sdk/sdk/src/main/res/layout/ghs_item_payment_provider_app.xml (hunk 1 above)
Comment (danicretu) [reply health-sdk/sdk/src/main/res/layout/ghs_item_payment_provider_app.xml:11]: Switched to `FrameLayout`, thank you for your suggestion.

File: health-sdk/sdk/src/main/res/drawable/ghs_bordered_background.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--TODO to be merged with same drawable from branch IPC 160-->
+<shape android:shape="rectangle" xmlns:android="http://schemas.android.com/apk/res/android">
+    <corners android:radius="@dimen/ghs_small"/>
+    <stroke android:color="@color/ghs_payment_input_background_color" android:width="1dp"/>
+    <solid android:color="@color/ghs_payment_input_border_color"/>
```
Comment (a-szotyori) [inline health-sdk/sdk/src/main/res/drawable/ghs_bordered_background.xml:6]: [Request] We should avoid using colors meant for a certain role in a different place. Can you please create a dedicated color for the background, maybe called `ghs_bordered_background_color`?

File: health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt [hunk 3]
```diff
@@ -42,7 +42,9 @@ class PaymentComponentView(context: Context, attrs: AttributeSet?) : ConstraintL
             }
         }
 
-    private val binding = GhsViewPaymentComponentBinding.inflate(getLayoutInflaterWithGiniHealthTheme(), this)
+    private val binding = GhsViewPaymentComponentBinding.inflate(getLayoutInflaterWithGiniHealthTheme(), this).also {
+        it.ghsSelectBankPicker.root.background = context.getDrawable(R.drawable.ghs_bordered_background)
```
Comment (a-szotyori) [inline health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt:46]: [Request] AndroidStudio recommended not to use `context.getDrawable()` directly and we should make sure we apply our theme to the context:
```suggestion
        it.ghsSelectBankPicker.root.background = ContextCompat.getDrawable(context.wrappedWithGiniHealthTheme(), R.drawable.ghs_bordered_background)
```

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Awesome work! I have only some small wishes. 🤓

File: health-sdk/sdk/src/main/res/drawable/ghs_bordered_background.xml (hunk 2 above)
Comment (danicretu) [reply health-sdk/sdk/src/main/res/drawable/ghs_bordered_background.xml]: Changed to suggested naming.

File: health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt (hunk 3 above)
Comment (danicretu) [reply health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt]: Thank you for pointing this out - I switched to using wrapped context.

**Review by a-szotyori — APPROVED** [review]
Thank you!

**Pull request merged**